reference/sql/statements/create-sequence.md

```diff
@@ -270,7 +274,19 @@ ERROR 4135 (HY000): Sequence 'test.seq2' has run out
 
 ## MySQL 兼容性
 
-* MySQL 暂无 SEQUENCE 选项。TiDB Sequence 借鉴自 MariaDB，但是 setval 会保持原有的步调。
+MySQL 暂无 SEQUENCE 选项。TiDB SEQUENCE 借鉴自 MariaDB，但是 `SETVAL` 会保持原有的步调。
+
+这里的步调是指，SEQUENCE 序列中的数在定义之后会产生一定的等差关系。`SETVAL` 虽然可以将 SEQUENCE 的当前值进行移动设置，但是后续出现的值仍会遵循原有的等差关系。
+
+示例如下：
+
+```
+1, 3, 5, ...            // 序列遵循起始为 1、步长为 2 的等差关系。
+select setval(seq, 6)   // 设置 SEQUENCE 的当前值为 6。
+7, 9, 11, ...           // 后续产生值仍会遵循这个等差关系。
+```
```

Comment:
Please use some SQLs to indicate this.
e.g. select nextval...(show output) ; select setval ....; select nextval ...(show output).

Reply:
here is ok, this wants to explain the logic of the numbers, using the SQL show here will be a huge page.